### PR TITLE
Make it compatible with Node 12 / Electron 5

### PR DESCRIPTION
**Pull request by aabuhijleh**
* Update "cld" dependency which was incompatible with Node 12 / Electron 5 and prevented successful compilation.

* Fix spellchecking on Windows

### CODE REVIEW

**Comment by quanglam2807** [comment]
Please merge this PR.

File: package.json [hunk 1]
```diff
@@ -27,11 +27,11 @@
   "main": "src/index.js",
   "homepage": "https://github.com/electron-userland/electron-spellchecker",
   "dependencies": {
-    "@nornagon/cld": "^2.4.9",
     "@felixrieseberg/spellchecker": "^4.0.10",
     "bcp47": "^1.1.2",
+    "cld": "^2.5.1",
     "debug": "^4.1.1",
-    "electron-remote": "^1.3.0",
+    "electron-remote": "https://github.com/aabuhijleh/electron-remote/tarball/master",
```
Comment (felixrieseberg) [inline package.json]: I don't think that solution is quite enough to fix things here - it's worthy hack, but I'd rather have a real fork than pure GitHub repo. 

Would you mind publishing your fork as an (immutable) fork to npm?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "electron-spellchecker",
-  "version": "2.1.0",
+  "version": "2.2.0",
```
Comment (felixrieseberg) [inline package.json]:
```suggestion
  "version": "2.1.0",
```

Let's leave this at the current version, we'll bump this right after!

File: package-lock.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "electron-spellchecker",
-  "version": "2.1.0",
+  "version": "2.2.0",
```
Comment (felixrieseberg) [inline package-lock.json]:
```suggestion
  "version": "2.1.0",
```

(See comment below)

File: package.json (hunk 2 above)
Comment (aabuhijleh) [reply package.json]: done.

File: package.json (hunk 1 above)
Comment (aabuhijleh) [reply package.json]: I agree with you that this is a hack. Maybe "electron-remote" should be replaced all together (#152).

For now, I tested this change on my Electron app on Windows and it's working well.

**Pull request merged**